tests/chainer_tests/functions_tests/loss_tests/test_huber_loss.py

```diff
@@ -96,8 +96,14 @@ def test_backward_gpu(self):
 
     def check_double_backward(self, x_data, t_data, y_grad, x_grad_grad,
                               t_grad_grad):
+
+        delta = 1
+        mask = numpy.abs(numpy.abs(x_data - t_data) - delta) < 1e-2
```

Comment:
If you'd like to avoid singular points manually, RHS could be `self.double_backward_options['eps']`.

Reply:
Fixed.